cleanlab/internal/numerics.py

```diff
@@ -0,0 +1,37 @@
+from typing import Optional
+import numpy as np
+
+
+def softmax(
+    x: np.ndarray, temperature: float = 1.0, axis: Optional[int] = None, shift: bool = True
```

Comment:
Should this default shift be `True` or `False`? 
Imo I think it's better to leave it as false by default and then pass in `shift=True` when intended

Reply:
Changed.